.circleci/config.yml

```diff
@@ -726,6 +726,46 @@ workflows:
           suite: itest-dup_mpool_messages
           target: "./itests/dup_mpool_messages_test.go"
       
+      - test:
+          name: test-itest-eth_account_abstraction
+          suite: itest-eth_account_abstraction
+          target: "./itests/eth_account_abstraction_test.go"
+      
```

Comment:
Most of those tests are very short, and instantiating separate runner for each is probably very overkill (and expensive in terms of CI credits). Easiest fix would be to just throw those into a single file, tho maybe not what we want to do

(this can be addressed later)

Reply:
The granularity is too fine, but we're letting these tests develop independently. There are several folks working on them and this strategy reduces conflicts. We can then go and consolidate them pretty easily.